### ISSUE TITLE
Unable to edit a project

### ISSUE THREAD

**Issue by connorhsm** [voteflux/flux-projects#19]
**Is your feature request related to a problem? Please describe.**
There is no method to edit project data if you make a typo when creating one or something has changed about the project e.g. a new leader.

**Describe the solution you'd like**
A new command to edit projects
`!edit [ID]`

**Describe alternatives you've considered**
Confirmation of the data you've entered before it is submitted to the database.
While this may work well for dates, which can be confusing due to many different preferred formats, it is mostly redundant as you can still view the history of what you have submitted in your private message chat history with the bot, unlike a traditional web form.

We require a way to edit projects regardless, confirmation before submitting the data may be an additional feature down the road if the need arises.

**Additional context**
None

**Comment by connorhsm** [comment]
User flow:
- User runs `!edit [ID]`
- Check if user has role `Flux Vetted` or is the Project lead for this specific project
- User asked in a PM which fields they wish to edit (multi-choice question)
- User asked in a PM for the data for each field they indicated to edit and provided the existing value
- Values updated and success message sent to user

**Comment by connorhsm** [comment]
Struggling to progress with this.
How can I retrieve a past response to a question (eg, the title of a project) and know what we asked them so that we can then ask them this again to update the response?

Light bulb moment.
I'm going about this the wrong way. It shouldn't be: ask a question and receive a response, everything should be a field that the user fills in. This isn't about surveying (asking questions) this is about collecting data (filling a form).

I recall almost coming upon this back when I was creating the questioning backend. Look at the questions, there is no substance to them. 

- What is your project title?
- What is the start date of your project?
- What is the end date of your project?
- What is the desciption of your project?
- What are the outcomes of your project?
- What are the deliverables of your project?
- What is the objective of your project?

They all follow a very similar template, requesting something, what that something is and a few adjustments for phrasing/context. Let's strip all these unnecessary words out and only say it once. "Please fill in the following fields about your new project: Title, Description etc". These become the labels for each data point and respectively, the column names in the database, solving this issue.

TLDR: We need to focus on filling data, like a form and make sure it's not like a survey where we ask questions.
